src/reference/asciidoc/graph.adoc

```diff
@@ -251,6 +257,9 @@ The preceding gateway produces nodes similar to the following:
 You can use this  `IntegrationNode` hierarchy for parsing the graph model on the client side as well as to understand the general Spring Integration runtime behavior.
 See also <<./overview.adoc#programming-tips,Programming Tips and Tricks>> for more information.
 
+The version 5.3 has introduced an `IntegrationPattern` abstraction and all out-of-the-box components, which belongs to particular EI pattern implements this abstraction and provides an `IntegrationPatternType` enum value.
+This information can be useful for some categorizing logic in the target application or being exposed into the graph node it can be used on UI to determine how to draw the provided component.
+
 === Integration Graph Controller
```

Comment:
Merged as 71224bc1c6738c1f4eb877c0d3d4763c94b0eb02 with minor doc polishing and added new elements to the graph example above this paragraph.